addons/mail/static/src/new/composer/emoji_picker.js

```diff
@@ -89,7 +89,6 @@ export class EmojiPicker extends Component {
     selectCategory(ev) {
         const id = Number(ev.target.dataset.id);
         if (id) {
-            this.state.categoryId = id;
```

Comment:
I think you should keep that, otherwise the highlighted category is not updated until scroll.
![bug](https://user-images.githubusercontent.com/6569390/203523276-fd81593c-7d60-4e09-9a33-8b66a04c5467.gif)